css/general.css

```diff
@@ -1,8 +1,55 @@
+
 body {
+  max-width: 1200px;
```

Comment:
Do not set max-width to the body. If you need a wrapper than controls your content's width - you should set max-width + width + margin: 0 auto. to this wrapper.

Example:

```
<body>
  <div class="main-wrapper">
  </div>
```
....

```
.main-wrapper {
  width: 1200px;
  max-width: 95%;
  margin: 0 auto;
```